### DISCUSSION TITLE
podcast plugin - cannot delete items from list (Version: 9.0.0 - 1714490454)

### DISCUSSION THREAD

**Discussion by dierochade** [LMS-Community/slimserver#1079] [General]
I run Lyrion Music Server Version: 9.0.0 - 1714490454 

In settings → podcasts → I can add new sources, but cannot delete. The delete button ('Löschen') seems just plain text and no action is executed.

![image](https://github.com/LMS-Community/slimserver/assets/125600882/d41061e3-ccd5-4c2b-b182-7c4b52ed21c6)

**Comment by michaelherger** [comment]
Check the box, then apply the changes with the button bottom right (not shown in your screenshot).

**Reply by dierochade** [reply]
ok, thank you very much. 
In this point, I apparently did not get the concept of the ui...
